### PR TITLE
Feature/e commerce template Clara

### PR DESCRIPTION
**Pull request by CRR257**

### CODE REVIEW

File: staff/clara-rufi/e-commerce-template/e-commerce.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+           <div class= "rightnav">
+            <a href="#contact">Contact</a>
+            <a href="#about">About</a>
+            <i class="fas fa-bars hamburguer"></i>
+        </div>  
+     
+    </header>
+
+    <main>
+        <h1>Bike selection</h1>
```
Comment (maryammdot) [inline staff/clara-rufi/e-commerce-template/e-commerce.html:36]: Maybe I would put the title h1 inside the section

File: staff/clara-rufi/e-commerce-template/ecommerce.css [hunk 2]
```diff
@@ -0,0 +1,121 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto');
+
+body{
+    margin: 0px;
```
Comment (maryammdot) [inline staff/clara-rufi/e-commerce-template/ecommerce.css:4]: if you set it to 0 you don't need to ad px. you could write margin: 0;

File: staff/clara-rufi/e-commerce-template/e-commerce.html (hunk 1 above)
Comment (CRR257) [reply staff/clara-rufi/e-commerce-template/e-commerce.html:36]: ok, thanks for this suggestion.

File: staff/clara-rufi/e-commerce-template/ecommerce.css (hunk 2 above)
Comment (CRR257) [reply staff/clara-rufi/e-commerce-template/ecommerce.css:4]: ok, thanks

**Pull request merged**